android/services/mongodb-remote/src/main/java/com/mongodb/stitch/android/services/mongodb/remote/RemoteMongoCollection.java

```diff
@@ -240,4 +240,11 @@
       final Bson filter,
       final Bson update,
       final RemoteUpdateOptions updateOptions);
+
+  /**
+   * A set of synchronization related operations at the collection level.
```

Comment:
Still needs a comment update per last review

Reply:
I must've missed that comment.